redash/destinations/microsoft_teams_webhook.py

```diff
@@ -0,0 +1,79 @@
+import logging
+import requests
+
+from redash.destinations import *
+from redash.utils import json_dumps
+from redash.serializers import serialize_alert
+
+
+class MicrosoftTeamsWebhook(BaseDestination):
+    @classmethod
+    def name(cls):
+        return "Microsoft Teams Webhook"
+
+    @classmethod
+    def type(cls):
+        return "microsoft_teams_webhook"
+
+    @classmethod
+    def configuration_schema(cls):
+        return {
+            "type": "object",
+            "properties": {
+                "url": {"type": "string"},
+            },
+            "required": ["url"],
+        }
+
+    @classmethod
+    def icon(cls):
+        return "fa-bolt"
+
+    def notify(self, alert, query, user, new_state, app, host, options):
+        """
+        :type app: redash.Redash
+        """
+        try:
+            data = {
```

Comment:
What about supporting custom templates?

Reply:
I can try to copy what the chatwork destination did for message templates.

Reply:
<img width="520" alt="Screen Shot 2022-01-27 at 10 19 36 AM" src="https://user-images.githubusercontent.com/3395738/151427642-98403cc5-6420-48d9-bbad-74883d6be835.png">